pkgs/top-level/all-packages.nix

```diff
@@ -1416,6 +1416,8 @@ in
 
   gopass-jsonapi = callPackage ../tools/security/gopass/jsonapi.nix { };
 
+  git-credential-gopass = callPackage ../tools/security/gopass/git-credential.nix { };
```

Comment:
I think this should be under `gitAndTools`.

Reply:
just the call or should I also move the .nix file?
I put it there because it only makes sense for people using gopass anyway

Reply:
@benneti Actually, nvm we killed it about a month ago https://github.com/NixOS/nixpkgs/pull/109396